console/v1/types_console_quick_start.go

```diff
@@ -0,0 +1,132 @@
+	// +optional
+	Icon string `json:"icon,omitempty"`
+	// tags is a list of strings that describe the Quick Start.
+	// +optional
+	Tags []string `json:"tags,omitempty"`
+	// durationMinutes describes approximately how many minutes it will take to complete the Quick Start.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Minimum=1
+	// +required
+	DurationMinutes int `json:"durationMinutes,omitempty"`
+	// description is the description of the Quick Start. (includes markdown)
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=256
+	// +required
+	Description string `json:"description,omitempty"`
+	// prerequisites contains a list of Quick Start names that must be completed before this Quick Start.
+	// +optional
+	Prerequisites []string `json:"prerequisites,omitempty"`
+	// introduction describes the purpose of the Quick Start. (includes markdown)
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Introduction string `json:"introduction"`
+	// tasks is the list of steps the user has to perform to complete the Quick Start.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinItems=1
+	// +required
+	Tasks []ConsoleQuickStartTask `json:"tasks"`
+	// conclusion sums up the Quick Start and suggests the possible next steps. (includes markdown)
+	// +optional
+	Conclusion string `json:"conclusion,omitempty"`
+	// nextQuickStart is a list of the following Quick Starts.
+	// +optional
+	NextQuickStart []string `json:"nextQuickStart,omitempty"`
+	// accessReviewResources contains a list of resources that the user's access
+	// will be reviewed against in order for the user to complete the Quick Start.
+	// The Quick Start will be hidden if any of the access reviews fail.
+	// +optional
+	AccessReviewResources []authorizationv1.ResourceAttributes `json:"accessReviewResources,omitempty"`
+}
+
+// ConsoleQuickStartTask is a single step in a Quick Start.
+type ConsoleQuickStartTask struct {
+	// title describes the task and is displayed as a step heading.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Title string `json:"title"`
+	// description describes the steps needed to complete the task. (includes markdown)
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Description string `json:"description"`
+	// review contains instructions to validate the task is complete. The user will select 'Yes' or 'No'.
+	// using a radio button, which indicates whether the step was completed successfully.
+	// +optional
+	Review *ConsoleQuickStartTaskReview `json:"review,omitempty"`
+	// recapitulation contains information about the passed step.
+	// +optional
+	Recapitulation *ConsoleQuickStartTaskRecapitulation `json:"recapitulation,omitempty"`
```

Comment:
Recapitulation isn't going to be understood by most people i don't think.  Result? Summary?

Reply:
I could go with `Summary` although I went with `Recapitulation` since its doing a recap of the passed step.
@spadgett thoughts?

Reply:
Summary is the more common word. I'd vote for that.

Reply:
will change it ! :)